corehq/ex-submodules/phonelog/management/commands/migrate_device_entry.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import print_function
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from phonelog.models import OldDeviceReportEntry, DeviceReportEntry
+
+COLUMNS = (
+    "xform_id", "i", "msg", "type", "date", "server_date", "domain",
+    "device_id", "app_version", "username", "user_id"
+)
+
+
+class Command(BaseCommand):
```

Comment:
before we run this we may also want to just double check we have enough space to fit it